crates/bevy_sprite/src/sprite.rs

```diff
@@ -240,41 +241,31 @@ pub enum ScalingMode {
     FitEnd,
 }
 
-/// How a sprite is positioned relative to its [`Transform`].
-/// It defaults to `Anchor::Center`.
-#[derive(Component, Debug, Clone, Copy, PartialEq, Default, Reflect)]
+/// Normalized (relative to its size) offset of a 2d renderable entity from its [`Transform`].
+#[derive(Component, Debug, Clone, Copy, PartialEq, Deref, DerefMut, Default, Reflect)]
```

Comment:
What do you think about doing an explicit Default impl which returns `CENTER` instead of relying on the `Vec2` default value? I know they are the same but it feels less obscure to me.

Reply:
Yeah that's a good idea I think, it's better to be explicit in this case.